ci-operator/config/openshift/release/openshift-release-master__nightly-4.10.yaml

```diff
@@ -362,6 +362,15 @@ tests:
     - chain: ipi-gcp-pre
     - ref: fips-check
     workflow: openshift-e2e-gcp
+- as: e2e-openstack-3nodes
+  interval: 72h
+  steps:
+    cluster_profile: openstack-vexxhost
+    env:
+      OPENSHIFT_INSTALL_MASTERS_SCHEDULABLE: "true"
+      TEST_SUITE: experimental/reliability/minimal
+      WORKER_REPLICAS: "0"
```

Comment:
Instead of setting `WORKER_REPLICAS` to 0, it's probably best to define a new variant called `compact` for consistency with other platforms.

Take for example:
https://github.com/openshift/release/blob/f85d035/ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-ref.yaml#L13-L25

Reply:
I also was looking for consistency: https://github.com/openshift/release/search?q=WORKER_REPLICAS

`SIZE_VARIANT` seems more related to flavors sizes, than the number of nodes, isn't?

Reply:
It control both the flavor size and the number of nodes:
https://github.com/openshift/release/blob/f85d0359ead283b0b0d0337590ff290f519549cf/ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh#L29-L40

I think that's what we should be aiming for.

As far as I can see, AWS, GCP, and Azure jobs use the compact SIZE_VARIANT while only libvirt uses WORKER_REPLICAS.

Reply:
done